### ISSUE TITLE
Deprecated "File.exists?" method has been removed in Ruby 3.2 in favor of "File.exist?"

### ISSUE THREAD

**Issue by jakevose** [brenes/stopwords-filter#27]
Hi, please see:

https://github.com/brenes/stopwords-filter/blob/4682dc5b32e8ba2a511ea19732bee5438c93770a/lib/stopwords/snowball/filter.rb#L11

It's causing "NoMethodError: undefined method `exists?' for File:Class"

**Comment by mgmodell** [comment]
This is true, but it seems this repo is no longer maintained. See discussion in #23 for others who are picking it up.
